code/modules/mob/living/living.dm

```diff
@@ -1968,6 +1968,10 @@
 /mob/living/proc/will_escape_storage()
 	return FALSE
 
+//Used specifically for the clown box suicide act
+/mob/living/carbon/human/will_escape_storage()
```

Comment:
I'm confused, what makes the clown box suicide different from the normal box suicide in which you'd need this? I like the heads staying in the boxes, it's funny

Reply:
The whole body goes into the storage, not the head. The regular boxes take the head, but the clown box takes the body. This is just so you can't pick up the mob_holder and have a whole human in the palm of your hand